src/Appwrite/Extend/Exception.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Appwrite\Extend;
+
+class Exception extends \Exception
+{
+    /**
+     * Error Codes
+     *  
+     * Naming the error types based on the following convention 
+     * <ENTITY>_<ERROR_TYPE>
+     * 
+     * Appwrite has the follwing entities:
+     * - General
```

Comment:
One last doubt, Queries and OAuth are an exception in this list. All the other are resources while this two are not really part of this category. Should we treat them differently? WDYT?

Reply:
We've now moved them over to General and projects 👍